### PR TITLE
monaco config updates for app-hipstershop

### LINKED ISSUE #380: [BUG] Hipstershop synthetic clickpath monaco fix
Hipstershop click path Monaco configurations are not aligned with the application UI components. A new click path must be defined.

### PR DESCRIPTION
**Pull request by eemrdog**

### CODE REVIEW

File: user-skel/ansible/roles/app-hipstershop/tasks/apply-dt-configuration.yml [hunk 1]
```diff
@@ -41,6 +42,27 @@
     tasks_from: source-node-id
   when: skip_synthetic_monitor == "false" and dt_synthetic_node_id is not defined 
 
+
+# - name: Source Host Group ID
```
Comment (tobigremmer-dt) [inline user-skel/ansible/roles/app-hipstershop/tasks/apply-dt-configuration.yml]: Please remove if not needed.

File: user-skel/ansible/roles/app-hipstershop/files/monaco/projects/infrastructure/conditional-naming-service/NamespacePod.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
  "displayName": "{{.name}}",
  "enabled": true,
- "nameFormat": "{ProcessGroup:KubernetesContainerName}.{ProcessGroup:KubernetesNamespace}",
+ "nameFormat": "{ProcessGroup:KubernetesNamespace}.{ProcessGroup:KubernetesBasePodName}",
```
Comment (tobigremmer-dt) [inline user-skel/ansible/roles/app-hipstershop/files/monaco/projects/infrastructure/conditional-naming-service/NamespacePod.json]: Please check: Does this align with our other examples/demos?

File: user-skel/ansible/roles/app-hipstershop/files/monaco/projects/hipstershop/synthetic-monitor/hipstershop-http.json [hunk 3]
```diff
@@ -1,66 +1,72 @@
...
-    ],
-    "anomalyDetection": {
-      "outageHandling": {
-        "globalOutage": false,
-        "localOutage": true,
-        "localOutagePolicy": {
-          "affectedLocations": 1,
-          "consecutiveRuns": 3
+  "name": "{{ .name }}",
+  "frequencyMin": {{.frequencyInMin}},
```
Comment (tobigremmer-dt) [inline user-skel/ansible/roles/app-hipstershop/files/monaco/projects/hipstershop/synthetic-monitor/hipstershop-http.json]: Small linting related comment: Please try to be consistent when using whitespace, naming variables, etc. For example: `{{.frequencyInMin}}` -> `{{ .frequencyInMin }}`

**Review by tobigremmer-dt — CHANGES_REQUESTED** [review]
Added a few comments. Rest looks good.

File: user-skel/ansible/roles/app-hipstershop/files/monaco/projects/infrastructure/conditional-naming-service/NamespacePod.json (hunk 2 above)
Comment (eemrdog) [reply user-skel/ansible/roles/app-hipstershop/files/monaco/projects/infrastructure/conditional-naming-service/NamespacePod.json]: Namespace - Pod Name based naming seemed to me more structured but the other examples use the ContainerName - NAmespace based naming. So, I will revert this back not to confuse the rest

File: user-skel/ansible/roles/app-hipstershop/tasks/apply-dt-configuration.yml (hunk 1 above)
Comment (eemrdog) [reply user-skel/ansible/roles/app-hipstershop/tasks/apply-dt-configuration.yml]: done

File: user-skel/ansible/roles/app-hipstershop/files/monaco/projects/hipstershop/synthetic-monitor/hipstershop-http.json (hunk 3 above)
Comment (eemrdog) [reply user-skel/ansible/roles/app-hipstershop/files/monaco/projects/hipstershop/synthetic-monitor/hipstershop-http.json]: ok

**Review by tobigremmer-dt — APPROVED** [review]
Looks good!

**Pull request merged**